### ISSUE TITLE
We don't use NPM or any module manager, so am getting "Cannot use import statement outside a module"....

### ISSUE THREAD

**Issue by natiki** [gruns/ImmortalDB#30]
Hi,

So I have a very simple page as follows:

```html
<!DOCTYPE html>
  
<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<title></title>
<script src="https://cdn.jsdelivr.net/npm/immortal-db@1.0.2/src/index.min.js"></script>
</head>
<body>
</body>
</html>
```

But now I get the following error in the console. 

![image](https://user-images.githubusercontent.com/4738140/66263014-81339f00-e837-11e9-9607-5ab184eec538.png)

1) So how do I use the library without any module manager etc.?
2) The rest of our application does not use module management amd I don't want to have to add one if I don't *really* have to. But if there is no other option can you provide a sample using my simple HTML page from above?

**Comment by natiki** [comment]
Doh! My bad.....

![image](https://user-images.githubusercontent.com/4738140/66263061-90671c80-e838-11e9-98a1-b9ae6f6df873.png)

I just took the default file on jsDeliver. However that points to **src** instead of *dist*. Once I point at *dist* all is well.

You can close this one assuming the default is correct and/or you want to correct the HTML escaping for the description on jsDelivr

**Comment by gruns** [comment]
Great catch. The default should be fixed.

I'll look into this. Or, if you know the solution, by all means submit a PR.

Thanks!

**Comment by gruns** [comment]
Circling back: this was fixed with ImmortalDB v1.0.3. jsdelivr now properly serves `/dist/immortal-db.min.js`.

  * https://www.jsdelivr.com/package/npm/immortal-db

Thank you, Donovan, for bringing this issue to my attention. Don't hesitate to let me know if there's anything else I can do for you.
